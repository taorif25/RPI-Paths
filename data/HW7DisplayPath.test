r
bad
2
r
2
bad
r
bad
bad
r
bad1
bad2
r
LINAC Facility
60
r
EMPAC
76
r
67
76
r
76
67
q