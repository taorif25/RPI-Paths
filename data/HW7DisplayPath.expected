First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [bad]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [bad]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [bad]
First building id/name, followed by Enter: Second building id/name, followed by Enter: Unknown building: [bad1]
Unknown building: [bad2]
First building id/name, followed by Enter: Second building id/name, followed by Enter: There is no path from LINAC Facility to Radio Club.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to EMPAC:
Total distance: 0.000 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from Academy Hall to EMPAC:
	Walk NorthWest to (Intersection 137)
	Walk West to (Cogswell Laboratory)
	Walk West to (Empire State Hall)
	Walk West to (MRC)
	Walk West to (EMPAC)
Total distance: 490.489 pixel units.
First building id/name, followed by Enter: Second building id/name, followed by Enter: Path from EMPAC to Academy Hall:
	Walk East to (MRC)
	Walk East to (Empire State Hall)
	Walk East to (Cogswell Laboratory)
	Walk East to (Intersection 137)
	Walk SouthEast to (Academy Hall)
Total distance: 490.489 pixel units.